### PR TITLE
Phark/blockchain additions

### PR DESCRIPTION
**Pull request by PharkMillups**

### CODE REVIEW

File: docs/blockchain/helium-token/helium-token.mdx [hunk 1]
```diff
@@ -5,4 +5,147 @@ sidebar_label: Helium Token
...
+At this point, the astute reader might be asking themselves the following question: 
+
+>
+If HNT supply is capped at 223,000,000 AND the Network is constantly burning HNT to mint Data Credits to enable transactions, won't we run out of HNT? 
+>
+
+**Yes.** This is where the idea of Net Emissions comes in. Along with with Max Supply, [HIP 20](https://github.com/helium/HIP/issues/73) also introduced the idea of Net Emissions. The [complete Net Emissions discussion in the HIP](https://github.com/helium/HIP/blob/master/0020-hnt-max-supply.md#net-emissions) is the best resource on this, but we'll provide a quick summary here:
+
+* In its simplest implementation, the blockchain would monitor how many HNT were burnt for Data Credits that epoch and adds them to the number of HNT to be minted that epoch. So in this simple implementation, if 10 HNT were burned for Data Credits in an epoch, the system would mint 3,434 HNT instead of 3,424 HNT.
+*
```
Comment (lthiery) [inline docs/blockchain/helium-token/helium-token.mdx:136]:
```suggestion

```

File: docs/blockchain/helium-token/helium-token.mdx [hunk 2]
```diff
@@ -5,4 +5,147 @@ sidebar_label: Helium Token
...
+
+>
+If HNT supply is capped at 223,000,000 AND the Network is constantly burning HNT to mint Data Credits to enable transactions, won't we run out of HNT? 
+>
+
+**Yes.** This is where the idea of Net Emissions comes in. Along with with Max Supply, [HIP 20](https://github.com/helium/HIP/issues/73) also introduced the idea of Net Emissions. The [complete Net Emissions discussion in the HIP](https://github.com/helium/HIP/blob/master/0020-hnt-max-supply.md#net-emissions) is the best resource on this, but we'll provide a quick summary here:
+
+* In its simplest implementation, the blockchain would monitor how many HNT were burnt for Data Credits that epoch and adds them to the number of HNT to be minted that epoch. So in this simple implementation, if 10 HNT were burned for Data Credits in an epoch, the system would mint 3,434 HNT instead of 3,424 HNT.
+* 
+*
```
Comment (lthiery) [inline docs/blockchain/helium-token/helium-token.mdx]:
```suggestion

```

**Review by lthiery — CHANGES_REQUESTED** [review]
just some dangling bullets but perhaps you were going to finish this before landing?

**Review by kent-williams — APPROVED** [review]
LGTM

**Review by kent-williams — APPROVED** [review]

**Review by lthiery — APPROVED** [review]

**Pull request merged**